requirements.txt

```diff
@@ -0,0 +1 @@
+ansible-core
```

Comment:
I was asking about this a lot in Slack, but the PyPI version of `ansible-core` doesn't seem to be good. But given the purpose of this file, I'm not sure if it would be a problem.

Reply:
I can demo more, but the way our build system works now in zuul with multi-stage builds, we use requirement files to pull in the needed bits.  I'll link build logs shortly.

Reply:
Should we just change this to a git url until this actually exists?

Reply:
we might, or we update makefile to git clone the repo. I can test tomorrow to see what happen if we use git+https://github here.

Reply:
Previously the thing we were doing was passing an Ansible branch into `make image` and it would install from source, from the `stable-2.9` or `devel` branch, or whatever.

We expect to do some version of that again, which may argue for the "git clone the repo" option, but I don't want to be opinionated beyond acknowledging the requirement.